packages/ngx-leaflet/ng-package.json

```diff
@@ -8,7 +8,7 @@
     "@acpaas-ui",
     "leaflet",
     "esri-leaflet",
-    "leaflet-draw",
-    "leaflet.vectorgrid"
+    "esri-leaflet-vector",
+    "leaflet-draw"
```

Comment:
Tip voor volgende keer: bij een opsomming onder elkaar zetten wij steeds ook een komma op de laatste regel. Zodoende zal bij een toevoeging ook enkel de aanpassing in de git diff verschijnen - lees: dan zie je properder alle veranderingen.

Reply:
![image](https://github.com/digipolisantwerp/antwerp-ui_angular/assets/46924202/fe8cfbd0-f3b4-49fd-b616-a3374f52a688)


Specifiek in de package.json lijkt het te zeggen dat het niet mag?

Reply:
Excuseer … je hebt gelijk. Dat kan alleen in gewoon ts en js files, niet in je (ng-)package.json - ignore my comment!